GMPWrapper/src/components/MarlinProcessorWrapper.cpp

```diff
@@ -157,8 +156,11 @@ std::shared_ptr<marlin::StringParameters> MarlinProcessorWrapper::parseParameter
       continue;
     }
 
-    parameterValues.push_back(parameterString);
+    auto split_parameter = gmp::util::split(parameterString);
+    for (auto& s : split_parameter) parameterValues.push_back(s);
```

Comment:
```suggestion
    parameterValues.insert(parameterValues.end(), split_parameter.begin(), split_parameter.end());
```

Reply:
I would argue that `.push_back()` is more specific (and efficient) here, since I always want to put it at the end of the vector. The insert has to do more checks, and as a developer I have to actually pause and check where is the developer putting the element.

Reply:
The insert has to check once for the range, the push back might cause allocation multiple times.
According to this https://stackoverflow.com/a/57521657 under their circumstances the insert is more efficient.

Add a comment for the developer.